docs/src/design/stack/field_ops.md

```diff
@@ -178,19 +178,25 @@ The effect on the rest of the stack is:
 * **No change** starting from position $1$.
 
 ## EXPACC
-The `EXPACC` operation pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
+The `EXPACC` operation computes one round of the expression $base^{exp}$. It is expected that `Expacc` is called at least `num_exp_bits` times, where `num_exp_bits` is the number of bits required to represent `exp`.
+
+It pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
```

Comment:
nit: feels like some redundancy with the word " top" in "It pops top $4$ elements from the top of the stack"

Reply:
ah yes, removed the first "top"